### PR TITLE
TOC Meeting Agenda 04 May 2023

### PR DESCRIPTION
**Pull request by arsulegai**
1. Initial version.

### CODE REVIEW

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+# Past due reports
+* 2023 Q1 Hyperledger Transact (**due 23 Feb 2023**)
+    * [Reminder to maintainers sent March 2](https://discord.com/channels/905194001349627914/941414458922790982/1080893688441491536)
+    * [Ongoing discussion with maintainers](https://discord.com/channels/905194001349627914/941414458922790982/1081247988275486752)
+    * [Waiting on list of maintainers from Shawn](https://discord.com/channels/905194001349627914/941414458922790982/1083097678461800580) to determine if they want to move project to Dormant
+    * Peter volunteered to open [issue](https://github.com/hyperledger/transact/issues/375) and [pull request](https://github.com/hyperledger/transact/pull/376) stating that the work is being moved to libsawtooth (opened on April 4, 2023)
+    * No update on issue or pull request as of April 11, 2023
+    * Arun reached out directly to Andrea and Shawn regarding the pull request, and it has since been merged
+    * Daniela will reach out to Shawn about status and moving to EOL
```
Comment (tkuhrt) [inline meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]:
```suggestion
    * Daniela will reach out to Shawn about status and moving to EOL
    * Shawn would like to move the project to Dormant state (see decision below)
    
* 2023 Q2 Hyperledger Sawtooth (**due 27 Apr 2023**)
```

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+# Upcoming reports
+* 2023 Q2 Hyperledger Sawtooth (due 27 Apr 2023)
```
Comment (tkuhrt) [inline meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]:
```suggestion
```
This is now overdue. Please send a reminder to the Sawtooth community and link above.

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+* 2023 Q2 Hyperledger Aries (due 04 May 2023)
+* 2023 Q2 Hyperledger Indy (due 04 May 2023)
+* 2023 Q2 Hyperledger AnonCreds (due 04 May 2023)
+* 2023 Q2 Hyperledger Iroha (due 11 May 2023)
+* [2023 TOC Project Update Calendar](https://wiki.hyperledger.org/display/TSC/2023+TOC+Project+Update+Calendar)
+
+# Discussion
+* Move Transact to Dormant.
```
Comment (tkuhrt) [inline meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]:
```suggestion
* Decision: [Move Transact to Dormant](https://github.com/hyperledger/toc/issues/98).
```

**Review by tkuhrt — CHANGES_REQUESTED** [review]

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md (hunk 1 above)
Comment (arsulegai) [reply meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]: Added the decision line.

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md (hunk 2 above)
Comment (arsulegai) [reply meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]: Done, shared the latest update.

File: meeting-minutes/2023/2023-05-04-TOC-meeting-record.md (hunk 3 above)
Comment (arsulegai) [reply meeting-minutes/2023/2023-05-04-TOC-meeting-record.md]: Updated, thanks.

**Comment by ryjones** [comment]
Please see #103

**Pull request closed without merging**